src/components/Table/Table.stories.tsx

```diff
@@ -105,3 +108,79 @@ export const ColumnChooser = {
     },
   },
 };
+
+export const Sorting = {
+  render: (args) => {
+    const { rowSize, isAlwaysVisible = [], isMultiSortable } = args;
+
+    const [sorting, setSorting] = useState<SortingState>();
```

Comment:
can we have multiple sorting examples with the isMultiple or document it well would work as well.

Reply:
check latest commit for documentation